packages/sdk-ts/src/core/modules/permissions/msgs/MsgUpdateActorRoles.spec.ts

```
@@ -0,0 +1,51 @@
+import { mockFactory, prepareEip712 } from '@injectivelabs/test-utils'
+import MsgUpdateActorRoles from './MsgUpdateActorRoles.js'
+import { IndexerGrpcWeb3GwApi } from './../../../../client'
+import { getEip712TypedData, getEip712TypedDataV2 } from '../../../tx/index.js'
+
+export interface PermissionRoleActors {
+  role: string
+  actors: string[]
+}
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)